backend/fleet_management/api.py

```diff
@@ -14,59 +14,48 @@
 
 
 class CurrentUserRetrieveView(views.APIView):
-
     def get(self, request):
-        return Response(
-            UserSerializer(request.user).data
-        )
+        return Response(UserSerializer(request.user).data)
 
 
 class PassengerListView(generics.ListAPIView):
     permission_classes = [GroupPermission]
     required_groups = all_driver_methods
     serializer_class = PassengerSerializer
     filter_backends = (filters.OrderingFilter, filters.SearchFilter)
-    search_fields = (
-        'first_name',
-        'last_name'
-    )
-    ordering = ('first_name', 'last_name')
+    search_fields = ("first_name", "last_name")
+    ordering = ("first_name", "last_name")
 
     def get_queryset(self):
         return User.objects.filter(
-            groups__name=Groups.Passenger.name,
-            country=self.request.user.country
+            groups__name=Groups.Passenger.name, country=self.request.user.country
         ) | User.objects.filter(country=None, groups__name=Groups.Passenger.name)
 
 
 class CarListView(generics.ListAPIView):
     permission_classes = [GroupPermission]
     required_groups = all_driver_methods
     serializer_class = CarSerializer
-    search_fields = ('plates',)
+    search_fields = ("plates",)
     filter_backends = (filters.OrderingFilter, filters.SearchFilter)
-    ordering = ('plates',)
+    ordering = ("plates",)
 
     def get_queryset(self):
-        return Car.objects.filter(
-            country=self.request.user.country,
-        )
+        return Car.objects.filter(country=self.request.user.country,)
 
 
 class DriveView(generics.ListCreateAPIView):
     permission_classes = [GroupPermission]
     required_groups = all_driver_methods
     serializer_class = DriveSerializer
     filter_backends = (filters.OrderingFilter,)
-    ordering = ('-date',)
+    ordering = ("-date",)
 
     def get_serializer_context(self):
-        return {'driver': self.request.user}
+        return {"driver": self.request.user}
 
     def get_queryset(self):
-        return Drive.objects.filter(
-            driver__id=self.request.user.id,
-        )
+        return Drive.objects.filter(driver__id=self.request.user.id,)
```

Comment:
that's unconnected with this pull request, but I'm pretty sure we could write that as:
```python
        return Drive.objects.filter(driver=self.request.user)
```